vochain/scrutinizer/scrutinizer.go

```diff
@@ -43,27 +32,25 @@ func (s *Scrutinizer) AddEventListener(l EventListener) {
 // Scrutinizer is the component which makes the accounting of the voting processes
 // and keeps it indexed in a local database.
 type Scrutinizer struct {
-	VochainState   *vochain.State
-	Storage        db.Database
-	votePool       []*models.Vote
-	processPool    []*types.ScrutinizerOnProcessData
-	resultsPool    []*types.ScrutinizerOnProcessData
-	entityCount    int64
+	VochainState      *vochain.State
+	votePool          []*models.Vote
+	newProcessPool    []*types.ScrutinizerOnProcessData
+	updateProcessPool [][]byte
```

Comment:
Can you explain a bit more the introduction of the `updateProcessPool` field on the commit msg ?

Reply:
Well, that's part of the badgerhold usage requirement, not sure adding it to the commit would be of use to anyone. I can instead add a comment in the code.

Reply:
Oki, yes would be nice to have it commented somewhere. As it is a requirement of badgerhold better on the code than on the commit like you said. Thanks